opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -109,6 +109,16 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
+  // Flags, a bit field. 8 least significant bits are the trace flags as
+  // defined in W3C Trace Context specification. 24 most significant bits are reserved
+  // and must be set to 0. Readers must not assume that 24 most significant bits
+  // will be zero.  When creating new spans, the most-significant 24-bits MUST be
+  // zero.  To read the 8-bit W3C trace flag (use flags & SPAN_FLAGS_TRACE_FLAGS_MASK).
+  // [Optional].
+  //
+  // See https://www.w3.org/TR/trace-context-2/#trace-flags for the flag definitions.
+  fixed32 flags = 16;
```

Comment:
For the record, why this is using field number 16 and not 17 as it is suggested on line 85 see this comment https://github.com/open-telemetry/opentelemetry-proto/pull/484/files#r1301972011

TLDR: field 16 was never used in a stable version of OTLP is is available for use.